src/serac/physics/contact_constraint.hpp

```diff
@@ -94,7 +94,11 @@ class ContactConstraint : public Constraint {
     // note: Tribol does not use cycle.
     int cycle = 0;
     contact_.update(cycle, time, dt);
-    return contact_.mergedGaps(false);
+    auto gaps_hpv = contact_.mergedGaps(false);
+    // TODO: this copy is needed to prevent the HypreParVector pointer from going out of scope.  see
```

Comment:
```suggestion
    // Note: this copy is needed to prevent the HypreParVector pointer from going out of scope.  see
```
If this is a TODO please add what the TODO is